PrebidMobileTests/AdUnitTests/AdUnitTests.swift

```diff
@@ -114,6 +114,98 @@ class AdUnitTests: XCTestCase {
         XCTAssertEqual(realBidInfo?.nativeAdCacheId, expectedCacheId)
     }
     
+    //forceSdkToChooseWinner + Winner = Contains Targeting Info
+    func testForcedWinnerAndWinningBid() {
+        //given
+        Targeting.shared.forceSdkToChooseWinner = true
```

Comment:
Please reset all global values to their default settings after the test completes (e.g. in [tearDown](https://developer.apple.com/documentation/xctest/xctest/1500463-teardown)) to prevent any unexpected behaviour.